src/main/java/com/alysaa/geyserupdater/common/util/GeyserProperties.java

```diff
@@ -30,10 +32,10 @@ public static boolean isLatestBuild() throws IOException {
      * @return the value of the property
      * @throws IOException if failed to load the Geyser git properties
      */
-    public static String queryGitProperties(String propertyKey) throws IOException {
-        Properties gitProp = new Properties();
-        gitProp.load(FileUtils.getResource("git.properties"));
-        return gitProp.getProperty(propertyKey);
+    public static String getGeyserGitPropertiesValueForPropertyKey(String propertyKey) throws IOException {
```

Comment:
I think `getGeyserGitPropertiesValue` would be better since we have a javadoc and keys are the only way to get a value

Reply:
Ah… yeah, that's just my own personal style when it comes to writing method/function names l o l

`doSomeTaskForArgumentName(WhateverType argumentName)`

Addressed by conforming to your stylistic choice in commit `5b4e9fa7b4aad55af9514accec6a2146d15c96e0`.